ee/clickhouse/models/property.py

```diff
@@ -225,55 +229,94 @@ def get_property_values_for_key(key: str, team: Team, value: Optional[str] = Non
     )
 
 
-def filter_element(filters: Dict, prepend: str = "") -> Tuple[str, Dict]:
+def process_ok_values(ok_values: Any, operator: OperatorType) -> List[str]:
```

Comment:
Nittiest nit: I like placing functions like this at the end of a file - more important stuff at the top. This doesn't really matter though :sweat_smile:

Reply:
I remember we talked about that at the offsite in October 😄 It's a fair point readability-wise